### ISSUE TITLE
Fade audio on pause/resume

### ISSUE THREAD

**Issue by Ambient-Impact** [jmshrv/finamp#799]
As someone who enjoys loud music, I find it way more pleasant to have it fade out quickly (like 200 ms or so?) and it feels less jarring or like I've hit a brick wall when I pause a track. I've used Poweramp on Android for years, and this is the one feature I miss switching over to Jellyfin and Finamp. Awesome work with the redesign by the way.

**Comment by Chaphasilor** [comment]
Hey, thanks!  
That should be possible to do, by simply quickly changing the volume, as suggested here: https://stackoverflow.com/questions/69508649/how-can-i-fade-in-and-fade-out-audio-in-just-audio

Not a priority for me, but if you or anyone else wants to tackle this, I'd be happy to accept a PR! It should be relatively easy to implement :)

**Comment by Ambient-Impact** [comment]
I'd love to if I had any experience working with any of this, but I'm a web developer and never worked on a native app; I know the limits of my knowledge far too well. 🤓

**Comment by Chaphasilor** [comment]
Dart is very similar to JavaScript, and you're not really touching native code. It would be similar to working with the Web Audio API in this case. You'd just need some OOP knowledge to understand how to overwrite methods of parent classes.

But no pressure! Just, feel free to ask if you'd to contribute :)

**Comment by Ambient-Impact** [comment]
True, Dart is pretty similar, and I do a ton of OOP stuff so that part I have a really good knowledge of. I'll think about it, though right now I'm short on time for these sorts of things because I have to gather bill repellent. 😭

**Comment by Chaphasilor** [comment]
@Ambient-Impact this will be added with the next beta update in a week or so!

**Comment by Chaphasilor** [comment]
Added in [beta update 0.9.16](https://github.com/jmshrv/finamp/releases/tag/0.9.16-beta)!

**Comment by Ambient-Impact** [comment]
@Chaphasilor Just installed the update and tried it out. I love it. Thanks!
